Making project compileable under linux (Permissions + Name Convention Violations)

DJGummikuh opened the pull request:
No structural changes yet, just preparing project for being build-able under linux (It's really amazing that the java compiler doesn't catch Inei <-> INEI in windows, even given the fact that Windows doesn't care... O.o)

Ellpeck commented:
Uhm.. the project should have already been compileable under linux I thought O_o
Please explain your change to the gradle file.

DJGummikuh commented:
I just changed the file permissions and added u+x (owner execute) so that I can actually use that command under linux to build your project by invoking "./gradlew install"
